example/basic/main.go

```diff
@@ -37,37 +33,15 @@ var (
 	anotherKey = kv.Key("ex.com/another")
 )
 
-// initTracer creates and registers trace provider instance.
-func initTracer() {
-	var err error
-	exp, err := tracestdout.NewExporter(tracestdout.Options{PrettyPrint: false})
-	if err != nil {
-		log.Panicf("failed to initialize trace stdout exporter %v", err)
-		return
-	}
-	tp, err := sdktrace.NewProvider(sdktrace.WithSyncer(exp),
-		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
-		sdktrace.WithResource(resource.New(kv.String("rk1", "rv11"), kv.Int64("rk2", 5))))
-	if err != nil {
-		log.Panicf("failed to initialize trace provider %v", err)
-	}
-	global.SetTraceProvider(tp)
-}
-
-func initMeter() *push.Controller {
-	pusher, err := metricstdout.InstallNewPipeline(metricstdout.Config{
-		Quantiles:   []float64{0.5, 0.9, 0.99},
-		PrettyPrint: false,
-	})
+func main() {
+	pusher, err := stdout.InstallNewPipeline([]stdout.Option{
+		stdout.WithQuantiles([]float64{0.5, 0.9, 0.99}),
+		stdout.WithPrettyPrint(),
+	}, nil)
 	if err != nil {
-		log.Panicf("failed to initialize metric stdout exporter %v", err)
+		log.Fatalf("failed to initialize stdout export pipeline: %v", err)
 	}
-	return pusher
-}
-
-func main() {
-	defer initMeter().Stop()
-	initTracer()
```

Comment:
This is a great example of what this unification is saving the user from. :smile: